SQL.js

```diff
@@ -1,5 +1,8 @@
 'use strict'
 const inspect = Symbol.for('nodejs.util.inspect.custom')
+const unsafe = Symbol.for('unsafe')
```

Comment:
```suggestion
const unsafe = Symbol('unsafe')
```

Let's not pollute the global registry with this.